Formula/swiftgen.rb

```diff
@@ -13,9 +13,12 @@ class Swiftgen < Formula
   end
 
   depends_on "ruby" => :build if MacOS.version <= :sierra
-  depends_on :xcode => ["9.2", :build]
+  depends_on :xcode => ["10.0", :build]
 
   def install
+    # Fix issue with libxml2, see https://github.com/Homebrew/brew/pull/4147
+    ENV["CC"] = Utils.popen_read("xcrun -find clang").chomp
```

Comment:
This isn't needed now with Xcode 10.

Removed in https://github.com/Homebrew/homebrew-core/pull/32684